internal/status/reporter.go

```diff
@@ -16,7 +16,7 @@ type StatusReporter interface {
 }
 
 func NewStatusReporter(logger logrus.FieldLogger, gql *graphql.Gql) StatusReporter {
-	if _, provided := os.LookupEnv(graphql.GqlProviderEndpointEnvKey); provided {
+	if _, provided := os.LookupEnv(graphql.GqlProviderIdentifierEnvKey); provided {
```

Comment:
If you think we should have additional env like `CONFIG_PROVIDER_ENABLED` or something like that, let me know 🙂